pkgs/tools/text/unoconv/unoconv-python3.patch

```diff
@@ -360,7 +360,7 @@ index 30e6706..f72cf08 100755
 -        print >>sys.stderr, "       http://github.com/dagwieers/unoconv/issues"
 +        print("unoconv: Cannot find a suitable office installation on your system.", file=sys.stderr)
 +        print("ERROR: Please locate your office installation and send your feedback to:", file=sys.stderr)
-+        print("       http://github.com/dagwieers/unoconv/issues", file=sys.stderr)
++        print("       https://github.com/dagwieers/unoconv/issues", file=sys.stderr)
```

Comment:
That is the content of a patch. Seems not necessary to fix URL in the print statement.

Reply:
I decided to keep it in anyways, doesn't hurt if it's part of the "addition" in the patch :D (And we all know that GitHub does HTTPS)